test/DynamicFees.t.sol

```diff
@@ -149,6 +151,186 @@ contract TestDynamicFees is Test, Deployers, GasSnapshot {
         assertEq(_fetchPoolSwapFee(key), 123);
     }
 
+    function test_updateDynamicSwapFee_100PercentFee_AmountIn() public {
```

Comment:
is this applying a protocol fee? unclear from the test

and is it just a gas snapshot or is it just setting that the dynamic fee is set?

Reply:
no protocol fee, i made it more clear